app/models/incident_report.py

```diff
@@ -61,6 +65,7 @@ def flip_coin():
                 date=fake.date_time_between(start_date="-1y", end_date="now"),
                 duration=timedelta(minutes=randint(1, 30)),
                 agency=choice(agencies),
+                user=choice(users),
```

Comment:
So every report will have a user but it's possible not every user will have a report? Sounds good.